src/api/index.ts

```diff
@@ -246,80 +242,16 @@ router.post(
 router.post(
 	"/dedupInstallationQueue",
 	bodyParser,
-	async (req: Request, res: Response): Promise<void> => {
-
-		const limit = req.body?.limit || 10000;
-		const jobs = await queues.installation.getJobs(["active", "delayed", "waiting", "paused"]);
-		const foundJobIds = new Set<string>();
-		const duplicateJobs: Job[] = [];
-
-		// collecting duplicate jobs per installation
-		for (const job of jobs) {
-
-			// we only want to deduplicate a certain number of jobs
-			if (duplicateJobs.length >= limit) {
-				break;
-			}
-
-			// getJobs() sometimes seems to include a "null" job in the array
-			if (!job) {
-				continue;
-			}
-
-			if (foundJobIds.has(`${job.data.installationId}${job.data.jiraHost}`)) {
-				duplicateJobs.push(job);
-			} else {
-				foundJobIds.add(`${job.data.installationId}${job.data.jiraHost}`);
-			}
-		}
-
-		// removing duplicate jobs
-		await Promise.all(duplicateJobs.map((job: Job) => {
-			logger.info({ job }, "removing duplicate job");
-			job.remove();
-		}));
-
-		res.send(`${duplicateJobs.length} duplicate jobs killed with fire.`);
+	async (_: Request, res: Response): Promise<void> => {
+		res.send(`This endpoint doesn't do anything useful anymore`);
```

Comment:
Maybe we could just kill the whole endpoint?

Reply:
yes, kill the whole endpoint! It's not being called anymore, anyways.